packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -12,7 +12,7 @@ import { Flex } from '../Flex';
 import { TabsProps, TabItemProps, Primitive } from '../types';
 import { View } from '../View';
 
-const isTabsType = (child: any): child is React.Component<TabItemProps> => {
+const isTabsType = (child: any): child is React.ReactElement<TabItemProps> => {
```

Comment:
Just curious, why was this change necessary?

Reply:
It seems `React.Component` is the legacy class component

Reply:
If anyone is curious: https://www.arahansen.com/how-children-types-work-in-react-18-and-typescript-4/